apps/api/src/resources/Event/resolver.ts

```diff
@@ -173,4 +197,9 @@ export class EventResolver {
   protected async event(@Arg("id", () => Number) id: number): Promise<Event> {
     return this.repository.findOneOrFail({ id });
   }
+
+  @Query(() => [Event])
+  protected async eventsWithKey(@Arg("urlKey", () => String) urlKey: string): Promise<Event[]> {
+    return this.repository.find({ urlKey });
```

Comment:
if you want clients to sort the events themselves then document it, but i think you should fix the duplicates on insertion